tests/lib/uoscore/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(uoscore)
```

Comment:
do we really need tests to be part of Zephyr? isn't this library tested by its authors?

Reply:
Yes, the library is tested by the authors, but I'd prefer to have some tests executed by the Zephyr CI as well, so that we don't introduce regression to the Zephyr integration over time.